examples/mbedTLS/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-mbedtls)
+
+hunter_add_package(mbedtls)
+find_library(
```

Comment:
>Yes, I think users of mbedtls will be grateful. Also it's not about Hunter so can be contributed upstream.

Alright, but that might take some time. (Same for aws-iot-device-sdk-cpp as well #940 )
Looks like there is already an issue for the same, https://github.com/ARMmbed/mbedtls/issues/298

>But I don't know with what branch to start.

Please use `mbedtls-2.5.2` branch.